core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -22,7 +22,7 @@ import slick.jdbc.{GetResult, JdbcProfile}
 case class SubmissionRecord(id: UUID,
                             workspaceId: UUID,
                             submissionDate: Timestamp,
-                            submitterId: String,
+                            submitterEmail: String,
```

Comment:
having type of this field as `WorkbenchEmail` seems better

Reply:
this is a database object representation, types need to match database types (unless we want to write the custom serializers which I don't want to right now). The way it is matches everything else so I am going to leave it alone.